src/LoggerFactory.ts

```diff
@@ -10,12 +10,24 @@ export enum Level {
 }
 
 const checkIfLevelKey = (key: any): key is keyof typeof Level => Object.keys(Level).includes(key);
+const isEventBelowLevel = (event: Event, level: Level = Level.Debug): boolean => {
+  const levelKey = capitalize(event.level);
+  return level > Level[levelKey];
+}
+
+const setEvents = (events: Array<Event>, level: Level = Level.Debug) => {
```

Comment:
Why is it called `setEvents`? Should probably be named something similar to `provideToFactory`.